chef/cookbooks/neutron/recipes/server.rb

```diff
@@ -184,15 +182,14 @@
 
 crowbar_pacemaker_sync_mark "wait-neutron_db_sync"
 
-config_files = "--config-file /etc/neutron/neutron.conf --config-file #{plugin_cfg_path}"
-if node[:neutron][:networking_plugin] == "cisco"
-  config_files += " --config-file /etc/neutron/plugins/ml2/ml2_conf_cisco.ini"
-end
-
 execute "neutron-db-manage migrate" do
   user node[:neutron][:user]
   group node[:neutron][:group]
-  command "neutron-db-manage #{config_files} upgrade head"
+  command 'source /etc/sysconfig/neutron; \
```

Comment:
the writing of that file is in a "only_if suse" clause. so this needs to be added here as well imho..

do we need an explicit "notify" of this execute block from the template block that writes the sysconfig file? I'm unsure.